apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -21,7 +22,7 @@ import FunctionEditor from './addons/functionEditor';
 import initializeGenerator from './addons/cdoGenerator';
 import CdoMetricsManager from './addons/cdoMetricsManager';
 import CdoRendererGeras from './addons/cdoRendererGeras';
-import CdoRendererThrasos from './addons/cdoRendererThrasos';
+import {CdoRendererThrasos} from './addons/cdoRendererThrasos';
```

Comment:
is this import used?

Reply:
Yes, but we haven't change how we're using it. It's currently our default renderer and we register it here:
https://github.com/code-dot-org/code-dot-org/blob/ec08eab64f1070ea68a08ad50b2b46e9a8819065/apps/src/blockly/googleBlocklyWrapper.js#L273-L278

The export did change however. Previously, the file just had a single class that was exported as the default. Now we're calling `addInlineRowSeparators` on the original class (renamed by appending "Base" to the name) to create a new constant that gets exported. (see apps/src/blockly/addons/cdoRendererThrasos.js)

Reply:
ah I missed we were changing the import, not adding a new one